### PR TITLE
Update OWNERS file

### PR DESCRIPTION
**Pull request by savitharaghunathan**

### CODE REVIEW

**Comment by jwmatthews** [comment]
Is it possible to leave me on as an approver but remove me from reviewers?

It'd be good for me to retain access to help unblock things and correct issues when needed, but agree doesn't make sense to keep me in a rotation for being assigned as a reviewer of PRs as a regular thing.

**Review by dymurray — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by kaovilai — COMMENTED** [review]
nice

**Comment by kaovilai** [comment]
/cherry-pick oadp-1.0

Still getting old reviewers list there

**Pull request merged**
